pygmt/tests/test_datasets_earth_dist.py

```diff
@@ -0,0 +1,53 @@
+"""
+Test basic functionality for loading Earth distance to shoreline datasets.
+"""
+
+import numpy as np
+import numpy.testing as npt
+from pygmt.datasets import load_earth_dist
+
+
+def test_earth_dist_01d():
+    """
+    Test some properties of the Earth distance to shoreline 01d data.
+    """
+    data = load_earth_dist(resolution="01d")
+    assert data.name == "z"
+    assert data.attrs["description"] == "GSHHG Earth distance to shoreline"
+    assert data.attrs["units"] == "km"
+    assert data.attrs["horizontal_datum"] == "WGS84"
+    assert data.shape == (181, 361)
+    assert data.gmt.registration == 0
```

Comment:
Need to update to the new syntax in #3696, depending on which PR is merged first.